apps/studio/prisma/schema.prisma

```diff
@@ -109,11 +109,10 @@ enum ResourceType {
 }
 
 model User {
-  id            String  @id @default(cuid())
-  name          String
-  email         String  @unique
-  phone         String
-  preferredName String?
+  id    String  @id @default(cuid())
+  name  String
+  email String  @unique
+  phone String?
```

Comment:
i believe phone is a required field for us to contact them for urgent cases

Reply:
no feature now to add their phone numbers leh. means DB will be in a fake state when users onboard, no?

Reply:
We are going to manually onboard users, so ops will fill in